src/sinks/util/auto_concurrency/service.rs

```diff
@@ -317,12 +316,12 @@ mod tests {
                 // must be borrowed mutable with a non-static
                 // lifetime. Resolving it is more work than it's worth
                 // for this test, so the clippy lint allow remains above.
-                for i in 0..concurrent {
-                    reqs[i] = Some(svc.send(i < concurrent - 1).await);
+                for (i, req) in reqs.iter_mut().take(concurrent).enumerate() {
+                    *req = Some(svc.send(i < concurrent - 1).await);
```

Comment:
This is :fire: Thanks! You should probably drop the last part of the phrase about the clippy lint, since you *did* remove it.